documentation/getting-started.md

```diff
@@ -46,6 +46,21 @@ skydive analyzer [--conf etc/skydive.yml]
 skydive client
 {% endhighlight %}
 
+### Helm
```

Comment:
I would place this in this file in the kubernetes section
https://github.com/skydive-project/skydive.network/blob/master/documentation/deployment.md